src/mngt/ocf_mngt_cache.c

```diff
@@ -1340,21 +1385,18 @@ static void _ocf_mngt_attach_load_superblock_complete(void *priv, int error)
 	ocf_pipeline_next(context->pipeline);
 }
 
-static void _ocf_mngt_attach_load_superblock(ocf_pipeline_t pipeline,
+static void _ocf_mngt_load_superblock(ocf_pipeline_t pipeline,
 		void *priv, ocf_pipeline_arg_t arg)
 {
 	struct ocf_cache_attach_context *context = priv;
 	ocf_cache_t cache = context->cache;
 
-	if (cache->device->init_mode != ocf_init_mode_load)
-		OCF_PL_NEXT_RET(context->pipeline);
-
 	ocf_cache_log(cache, log_info, "Loading cache state...\n");
 	ocf_metadata_load_superblock(cache,
-			_ocf_mngt_attach_load_superblock_complete, context);
+			_ocf_mngt_load_superblock_complete, context);
 }
 
-static void _ocf_mngt_attach_init_instance(ocf_pipeline_t pipeline,
+static void _ocf_mngt_attach_instance(ocf_pipeline_t pipeline,
```

Comment:
This name is not very informative. I think this function actually does two things: initializes cleaning policy and cleaner. So maybe splitting it into two pipeline steps would be the cleanest solution:
_ocf_mngt_init_promotion
_ocf_mngt_init_cleaner